apps/gateway/lib/gateway/controllers/curse_of_mirra/quest_controller.ex

```diff
@@ -44,4 +46,19 @@ defmodule Gateway.Controllers.CurseOfMirra.QuestController do
         |> send_resp(400, "Error completing bounty")
     end
   end
+
+  def add_quests(conn, %{"user_id" => user_id, "type" => type, "amount" => amount}) do
```

Comment:
Why do we have/need an endpoint to add quests? Who is to use this?

Reply:
It's still to be defined but we need to find a way to generate user quests when a user log in, so i though about generating quests through an endpoint to have something working. Maybe we could move it to the auth function or something like that but i think this could be developed when we finish this feature 
in https://github.com/lambdaclass/mirra_backend/issues/800